snap-css-web/tests/e2e/specs/clean.spec.js

```diff
@@ -0,0 +1,75 @@
+// https://docs.cypress.io/api/introduction/api.html
+/// <reference types="cypress" />
+
+function checkEditor(editor, expected) {
```

Comment:
is there a way to put this function somewhere it can be imported for all ?

Reply:
We can put it in one folder and provide the cy and editor object to avoid scope issues.